### PR TITLE
Render messages based on level

### PR DESCRIPTION
**Pull request by jcox-dev**
# Motivation and Context
Currently every message flashed is treated as an error. This PR adds the functionality to handle levels of messages and render the related panel type from the pattern library. 

# What has changed

- Adds functionality to respondent home for rendering non-error level messages in templates using the pattern library panels.
- Adds additional field "level" (currently only ERROR or INFO) to the message dicts.
- Adds an HTML partial for handling the message logic and rendering appropriately.
- Adds missing title tag to the base template.
- Use correct error message for an invalid code.

# How to test?
Unit tests:
`make unittests`

Functionality:
1. Start locally and attempt to navigate to respondent home: http://localhost:9092
2. Check the rendered messages against "Message Slides" (attached to Trello card).

image

NB: WIP as missing tests.

### CODE REVIEW

**Comment by jcox-dev** [comment]
[RH - Handle Info Messages](https://trello.com/c/YswGBlIx/288-rh-handle-info-messages)

File: app/templates/partials/messages.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </ul>
+      </div>
+      {% endfor %}
+    </div>
+
+    {% elif group=='INFO' %}
+      {% for message in messages %}
+      <div class="panel panel--simple panel--info">
+        <div class="panel__body">
+          {% for para in message.text.split('\n') %}
```
Comment (liamtoozer) [inline app/templates/partials/messages.html]: 👍

File: app/templates/base.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta content="" name="description">
   <meta content="width=device-width, initial-scale=1" name="viewport">
+  <title>{% block title %}My Survey{% endblock %}</title>
```
Comment (liamtoozer) [inline app/templates/base.html]: Does this need to be `My Study` to match the text from the banner?

Comment (jcox-dev) [reply app/templates/base.html]: Yes! My mistake.

File: app/templates/partials/messages.html (hunk 1 above)
Comment (jcox-dev) [reply app/templates/partials/messages.html]: Bit of a hack but it does what we need.

File: app/__init__.py [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 
-BAD_CODE_MSG = {'text': 'Please enter the 12 character access code provided to you by ONS.', "clickable": True}
-BAD_RESPONSE_MSG = {'text': 'There was an error, please enter your access code and try again.', "clickable": True}
-CODE_USED_MSG = {'text': 'Thank you. We have received a response for your address. No further action is required.\n\nIf you need further assistance, please phone us for free on 0800 085 7376.', "clickable": True}    # NOQA
-INVALID_CODE_MSG = {'text': 'Invalid access code. Please check your access code and web address to continue.', "clickable": True}  # NOQA
-NOT_AUTHORIZED_MSG = {'text': 'There was a problem connecting to this study. Please try again later.', "clickable": False}  # NOQA
+CODE_USED_MSG = {'text': 'Thank you. We have received a response for your address. No further action is required.\nIf you need further assistance, please phone us for free on 0800 085 7376.', "level": "INFO"}    # NOQA
+CLOSED_CE_MSG = {'text': 'This study is now closed. Thanks you for your interest in participating.', "level": "INFO"}  # NOQA
```
Comment (jrbarnes9) [inline app/__init__.py]: @JamesGardiner Typo: 'Thanks you...'

Comment (jcox-dev) [reply app/__init__.py]: @jrbarnes9 👋

**Pull request merged**